.github/workflows/build_all_apps.yml

```diff
@@ -262,6 +328,14 @@ jobs:
           build_path=$(echo '${{ env.build_path_list}}' |  sed -n 's/.*"${{ matrix.repo_name }}"[[:space:]]*:[[:space:]]*"\([^"]*\)".*/\1/p')
           echo "build_path=$build_path" >> $GITHUB_ENV
 
+      - name: Build for Stax
+        run: |
+          if echo "${{ env.build_for_stax_list }}" | grep -q "${{ matrix.repo_name }}" || [[ "${{ inputs.run_stax }}" ]]; then
```

Comment:
You can get rid of the `run_stax` argument I think :)

Reply:
`|| [[ "${{ inputs.run_stax }}" ]]` this part handle `run_stax` argument ;-)

Reply:
could be useful to check if an app build on Stax...

Reply:
I would argue that we have the app tester for this ?

Reply:
I prefer to keep it for now because it's easier to run directly the workflow with `run_stax `flag in order to see which app listed in SDK CI is Ok for build and could be added in `build_for_stax ` list